packages/core-sdk/test/integration/ipAccount.test.ts

```diff
@@ -159,4 +168,78 @@ describe("IPAccount Functions", () => {
     });
     expect(txHash).to.be.a("string").and.not.empty;
   });
+
+  it.only("should successfully transfer ERC20 tokens", async () => {
+    const erc20 = new Erc20Client(publicClient, walletClient);
+    // 1. Query token balance of ipId and wallet before
+    const initialErc20BalanceOfIpId = await erc20.balanceOf({
+      account: ipId,
+    });
+    const initialErc20BalanceOfWallet = await erc20.balanceOf({
+      account: TEST_WALLET_ADDRESS,
+    });
+    const initialWipBalanceOfIpId = await client.wipClient.balanceOf(ipId);
+    const initialWipBalanceOfWallet = await client.wipClient.balanceOf(TEST_WALLET_ADDRESS);
+    // 2. transfer erc20 token to the ip account
+    const txHash = await erc20.mint({
+      to: ipId,
+      amount: parseEther("0.002"),
+    });
+    await publicClient.waitForTransactionReceipt({
+      hash: txHash,
+    });
+    // 3. transfer wip to the ip account
+    await client.wipClient.deposit({
+      amount: parseEther("0.001"),
```

Comment:
can we use a smaller amount here (to save IP)?. I think any amount (ie 0-10) is okay to validate right?